src/openforms/js/components/admin/form_design/registrations/objectsapi/ObjectsApiOptionsFormFields.js

```diff
@@ -33,6 +40,12 @@ const ObjectsApiOptionsFormFields = ({index, name, schema, formData, onChange})
 
   const changeVersion = v => {
     const realVersion = v + 1;
+
+    if (realVersion === 1) {
+      const confirmV1Switch = window.confirm(v1SwitchMessage);
+      if (!confirmV1Switch) return;
+    }
```

Comment:
I could also not show the popup if `variablesMapping` is empty, but probably not necessary